components/NavBar.js

```diff
@@ -67,30 +79,31 @@ export default function NavBar() {
 
   return (
     <nav className={styles.navbar}>
-      <section className={styles.container}>
-        <div className={styles.logo}>
-          <Image src='/icon.png' alt='Course chart icon' height={50} width={50} />
-        </div>
-        <h1>CourseChart</h1>
-      </section>
+        <Link href='/' className={styles.homeLogo}>
+            <section className={styles.container}>
+              <div className={styles.logo}>
+                <Image src='/icon.png' alt='Course chart    icon' height={50} width={50} />
+              </div>
+              <h1>CourseChart</h1>
+            </section>
+      </Link>
 
       <section className={styles.courseButtons}>
         <Link href='/'>
-          <div className={styles.course}>
+          <div className={styles.course, styles.home}>
```

Comment:
I don't believe a comma separation for these classnames actually works, I think it only takes the first style. this should work though: `className={'${styles.course} ${styles.home}'}`
(the quotes inside the first set of brackets will need to be backticks but the formatting was weird for the comment so i made them single quotes)